src/oncall/api/v0/teams.py

```diff
@@ -86,7 +86,14 @@ def on_get(req, resp):
         query = '%s WHERE %s' % (query, where_query)
 
     cursor.execute(query, query_values)
-    data = [r[0] for r in cursor]
+    data = [None]
+    if 'get_id' in req.params:
```

Comment:
We can simplify logic here:
```
data = [r[0] for r in cursor]
if req.get_param_as_bool('get_id'):
  data = {} # fill in
``` 
Although I'm not sure what get_param_as_bool returns when the param doesn't exist (hopefully `None`).